app.vue

```diff
@@ -1,56 +1,153 @@
 <template>
-	<div>
-		<NuxtRouteAnnouncer />
-		<div class="wrapper">
-			<h1>evoize<span class="dot">.</span>me</h1>
-			<h2>Open-source invoicing & bill pay for freelancers.</h2>
-		</div>
-	</div>
-</template>
+	<div class="relative bg-gradient-to-t from-[#DBF1FE] to-[#FFFFFF]">
+		<AppNavigation />
 
-<style scoped>
-body {
-  background: #00091B;
-  color: #fff;
-}
+		<HeaderBackground />
+		<header class="container mx-auto flex flex-col px-0 sm:px-8">
+			<PageTeaser />
+			<div class="mt-20 flex flex-col lg:flex-row">
+				<PreviewCard
+					v-for="(card, index) in previewCards"
+					:key="index"
+					:card="card"
+				/>
+			</div>
+		</header>
 
-@keyframes fadeIn {
-  from { top: 20%; opacity: 0; }
-  to { opacity: 1; }
-}
+		<section
+			id="highlight"
+			class="mx-auto mt-20 max-w-7xl px-8 lg:mt-52"
+		>
+			<SectionTitle
+				title="Das Open Source Invoice Tool für Freelancer und kleine Unternehmen"
+			/>
+			<FeatureHighlight
```

Comment:
Suggestion: Rename to content card. This design is not only limited to highlight Features